liquibase-core/src/main/java/liquibase/database/core/Db2zDatabase.java

```diff
@@ -8,8 +8,8 @@
 import liquibase.util.StringUtil;
 
 public class Db2zDatabase extends AbstractDb2Database {
-	
-	public Db2zDatabase() {
+
+    Db2zDatabase() {
```

Comment:
Why did you remove the public from this? I think that will keep this extension from being loaded by the ServiceLoader, because it needs a public no-arg constuctor.